redbot/cogs/economy/economy.py

```diff
@@ -922,16 +927,19 @@ async def slottime(self, ctx: commands.Context, seconds: int):
         await ctx.send(_("Cooldown is now {num} seconds.").format(num=seconds))
 
     @economyset.command()
-    async def paydaytime(self, ctx: commands.Context, seconds: int):
+    async def paydaytime(self, ctx: commands.Context, seconds: TimedeltaConverter):
```

Comment:
Now that this accepts numbers with units, it would make more sense if this argument were named `duration` (or `time`, `cooldown`, whatever you think makes sense here).

Remember to also update the docstring with the new name and update this code:
```py
seconds = seconds.total_seconds()
```
to be this instead (or whatever name you choose for argument):
```py
seconds = duration.total_seconds()
```